GeneratorInterface/LHEInterface/src/LHEEvent.cc

```diff
@@ -306,7 +306,7 @@ std::auto_ptr<HepMC::GenEvent> LHEEvent::asHepMCEvent() const
 			genVertices.push_back(current_vtx);
 		}
 
-		for(unsigned int j = mother1; j <= mother2; j++)	// set mother-daughter relations
+		for(unsigned int j = mother1; j <= mother2; j++) {	// set mother-daughter relations
 			if (!genParticles.at(j)->end_vertex())
 				current_vtx->add_particle_in(genParticles.at(j));
 
```

Comment:
Let's keep the logic from the original code but just change the indentation to make the compiler happy.